runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java

```diff
@@ -169,6 +170,24 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		Collection<CrossbowmanStatue> statues = plugin.getStatues().values();
+		if (!statues.isEmpty() && config.highlightSepulchreStatues())
+		{
+			for (CrossbowmanStatue statue : statues)
+			{
+				if (statue.isAnimating() && statue.getGameObject() != null && statue.getGameObject().getClickbox() != null)
```

Comment:
`.getClickbox()` is a very expensive call, call it once and then store the result.

Reply:
It may be worth using `getConvexHull` instead actually since we generally try to avoid computing clickboxes for things you don't actually have to click, due to how slow it is.

Reply:
Did not know this. I took your advice here and used convex hull since it basically outlines the same thing.